src/SFA.DAS.VacancyServices.Functions/SchedulerFunctions/MonitorScheduler.cs

```diff
@@ -1,25 +0,0 @@
-using System;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Extensions.Logging;
-using SFA.DAS.VacancyServices.Functions.Infrastructure;
-
-namespace SFA.DAS.VacancyServices.Functions.SchedulerFunctions
-{
-    public class MonitorScheduler : SchedulerBase
-    {
-        public MonitorScheduler(StorageQueueService service)
-        : base(SchedulerFunctionConstants.QueueNames.MonitorSchedulerQueueName, service)
```

Comment:
Can probably remove this queue name from the constants file

Reply:
Done